lib/jazzy/doc_builder.rb

```diff
@@ -297,6 +297,13 @@ def self.document(source_module, doc_model, path_to_root)
         return document_index(source_module, path_to_root)
       end
 
+      tasks = render_tasks(source_module, doc_model.children)
+      tasks_list_children = tasks.map do |task|
+        if task[:name]
+          { name: task[:name], url: "#/"+task[:uid] }
+        end
+      end
```

Comment:
Just put it at the end of this line, otherwise the array you're mapping to can contain nil